packages/availity-react-kit/project/app/Request/api/requestApi.js

```diff
@@ -0,0 +1,71 @@
+import { defaults, get } from 'axios';
+import map from 'lodash.map';
+
+const ORGANIZATIONS_OPTS = 'api/sdk/platform/v1/organizations';
+const PROVIDERS_OPTS = 'api/internal/v1/providers';
+const USER_OPTS = 'api/sdk/platform/v1/users/me';
+const AUTHORIZATIONS_OPTS = 'api/v1/proxy/healthplan/v1/authorizations';
+
+export const getUser = () => {
```

Comment:
You can shorthand this:
```
export const getUser = () => get(USER_OPTS);
```
Same type of thing for the others which just return the first thing

Reply:
i'll update